src/nightwatch/setup-helper.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const fs = require('fs');
+const loadJsonFile = require('load-json-file');
+const globSync = require('glob').sync;
+
+const interatePackageDirectories = (lernaJSON) => {
+  const srcFolders = [];
+  loadJsonFile.sync(lernaJSON).packages.forEach((globPath) => {
+    globSync(globPath).forEach((packagePath) => {
+      const testPath = `${packagePath}/tests/nightwatch/`;
+      if (fs.existsSync(testPath)) {
+        srcFolders.push(testPath);
+
+        const files = fs.readdirSync(testPath);
+        files.forEach((file) => {
+          const subDir = path.join(testPath, file);
+          if (fs.statSync(subDir).isDirectory()) {
+            srcFolders.push(subDir);
+          }
+        });
+      }
+    });
+  });
+  return srcFolders;
+};
+
+const interateDirectories = (basePath) => {
```

Comment:
per my comment above, this may not be needed if we change our filter to use a glob pattern